rclcpp_action/src/types.cpp

```diff
@@ -41,6 +41,9 @@ to_string(const GoalUUID & goal_id)
 void
 convert(const GoalUUID & goal_id, rcl_action_goal_info_t * info)
```

Comment:
The function documentation should be updated to mention that it throws if `info` is `nullptr`. Same below.